business_requirement/models/business.py

```diff
@@ -67,6 +67,12 @@ def _get_default_company(self):
         states={'draft': [('readonly', False)],
                 'confirmed': [('readonly', False)]}
     )
+    terms_and_conditions = fields.Html(
+        'Terms and Conditions',
+        read_only=True,
```

Comment:
why readonly?
Shouldnot we add copy=true as well?

Reply:
default copy=True